protocol/src/main/java/io/zeebe/protocol/BpmnElementType.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.zeebe.protocol;
+
+public enum BpmnElementType {
+  UNSPECIFIED,
+  EVENT,
+  TASK,
+  SEQUENCE_FLOW,
+  GATEWAY,
+  PROCESS
```

Comment:
We need more fine granular element types. For example, to differentiate between a `startEvent`, a `intermediateCatchEvent` and a `endEvent`. A good idea could be to use the element names from the XML as element types.